stable/traefik/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: traefik
-version: 1.65.0
+version: 1.65.1
```

Comment:
please bump to since to is not a bug fix
```suggestion
version: 1.66.0
```

Reply:
Done

Reply:
@cpanato Can you have a look a the PR again?